packages/visualizations/src/components/Map/utils.ts

```diff
@@ -227,8 +229,78 @@ export const getFixedTooltips = (
     ) as ChoroplethFixedTooltipDescription[];
 };
 
-export const isVectorTile = (
-    shape: ChoroplethShapeValues
-): shape is ChoroplethShapeVectorTilesValue =>
-    (shape as ChoroplethShapeVectorTilesValue).layer !== undefined &&
-    (shape as ChoroplethShapeVectorTilesValue).key !== undefined;
+export const computeFilterExpression = (filterConfig: MapFilter) => {
+    /** Transform a filter object from the options into a Maplibre filter expression */
+    const { key, value } = filterConfig;
+    const filterMatchExpression: string[] = ['in', key];
+    (Array.isArray(value) ? value : [value]).forEach((filterValue) => {
+        filterMatchExpression.push(filterValue.toString());
+    });
+    return filterMatchExpression;
+};
+
+export const defaultFormat: ChoroplethTooltipFormatter = ({ value, label }) =>
```

Comment:
Maybe `defaultTooltipFormat` to be more explicit?

Reply:
👍